modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/FiltersDropdown.js

```diff
@@ -83,7 +83,6 @@ const FiltersDropdown = () => {
 						<ClayDropDown.ItemList>
 							{filters.map((filter) => (
 								<ClayDropDown.Item
-									active={filter.value !== undefined}
```

Comment:
`value` property of a filter doesn't seem to have any purpose in current code. This looks like some leftover junk code from previous implementation of filters menu :thinking: 

Another bug in the pile of "This would be avoided if we used TypeScript".